src/validate.js

```diff
@@ -502,6 +502,8 @@ function F_VALIDATE(disabledHL) {
 
 		this.$address = new _WV.SimpleADDRESS(attrs.primaryStreetID);
 
+		this.$isRoutable = this.$rawSegment.isRoutable();
```

Comment:
$isRoutable should be read only, i.e. set property "writable: false" as we do for other fields.